loadable_apps/loadable_sample/wifiapp/binary_update.c

```diff
@@ -669,6 +644,27 @@ void binary_update_test(void)
  ****************************************************************************/
 void binary_update_aging_test(void)
 {
-	binary_update_myself_test();
+	int ret;
+	binary_update_info_t bin_info;
+
+	while (1) {
+		ret = binary_update_getinfo(APP_NAME, &bin_info);
+		if (ret != OK) {
+			return;
+		}
+
+		/* Copy current binary with a higher version and valid crc. */
+		ret = binary_update_download_binary(&bin_info, DOWNLOAD_VALID_BIN);
+		if (ret != OK) {
+			return;
+		}
+
+		sleep(5);
+
+		ret = binary_update_reload(APP_NAME);
+		if (ret != OK) {
```

Comment:
Is it okay not to remove the downloaded binary? If not, it can consume the flash.

Reply:
Right. The new file needs to be removed in this case. I will modify it